aea/cli/run.py

```diff
@@ -195,35 +197,69 @@ def _profiling_context(period: int) -> Generator:
         sys.stderr = open(os.devnull, "w")
 
 
-def _print_hash_table(ctx: Context, aea: AEA) -> None:
-    """Print hash table of all available components."""
-    hash_data = []
-    ipfs_hash = IPFSHashOnly()
+def _print_instantiated_components(aea: AEA) -> None:
+    """Print table of only components."""
+    components: List[str] = []
     max_col_1_length = 0
-    max_col_2_length = 48
+
     for component_type in [
         ComponentType.PROTOCOL,
         ComponentType.CONNECTION,
         ComponentType.CONTRACT,
         ComponentType.SKILL,
     ]:
-        components = aea.resources.component_registry.fetch_by_type(component_type)
-        for component in components:
-            path = Path(
-                ctx.cwd,
-                VENDOR,
-                component.public_id.author,
-                component_type.to_plural(),
-                component.public_id.name,
-            )
-            if not path.exists():
-                path = Path(
-                    ctx.cwd, component_type.to_plural(), component.public_id.name
-                )
```

Comment:
What happened to this part? You have to account for this below.

Reply:
Since we're using glob approach we don't need to check for vendor and author packages separately.